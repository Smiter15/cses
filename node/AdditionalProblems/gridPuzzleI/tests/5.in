50
40 39 37 42 34 38 39 43 34 40 36 34 35 40 32 38 36 43 38 39 32 37 41 37 31 34 35 43 34 33 41 37 37 36 36 36 38 39 35 41 43 43 40 35 40 36 39 35 39 40
34 39 46 34 34 43 35 34 41 40 37 28 39 42 39 35 39 35 41 44 42 38 41 36 36 32 36 33 39 36 40 40 37 36 42 37 35 39 33 33 36 40 40 34 39 40 44 38 41 28
