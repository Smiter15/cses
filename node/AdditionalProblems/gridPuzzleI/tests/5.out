.XX..XX.XXX.XXXXXXXXXXXXX.X.XXXXXXXXXX..XXXXXXXXX.
.XX..X.XXXX.XXXXXXXXXXXXX.X.XXXXXXXXXX..XXX.XXXXX.
.XX.XXX.XXX.XXX.X.XXXXXXX.X.XXXXXXXX.X..XXX.XXXXX.
XXXX.X..XXX.XXX.X.XXXXXXX.XXXXXXXXXXXXXXXXXXXXXXX.
.XX.XXXXXXX.XXXXXXXXXXX.....X.XXX.XX.X...XX.XXXXX.
XXXX.X..XX..XXX.X.XXXXX....XXXXXXXXXXXXXXXXXXXXXX.
XXXXXXXXXXX.XXXXXXXXXXXXXXX.X.XX..X..X...XXXXXXXX.
.XX..X.XXXX.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.XXXXX.
XXXXXXX.XX..XXX.X.XXX.X.....X.XX..X.XXXXXXXXXXXXX.
..XXXXXXXXX.XXXXXXXXXXXXXXXXXXXXXXXX.X...XX.XXX.X.
XXX..X..XX...X....XXX.X.XXXXXXXXXXXXXXXXXXXXXXXXX.
XXXXXXXXXXX.XXXXXXXXXXXX......XX..X......XX.XXXXXX
..X..X..X....X..XXXXXXXXXXXXXXXXXXXXXXXXXXXX.XX.X.
XXXXXXXXXXXXXXXX..XXX.X.......XXXXXXXXXXXXXXXXXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXX....X...........X.X.
X.X..X.......X..XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXX...XX...........XXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..X.....XXXXXXXXXX
..X.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX......X...
XXXX.X..........XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXX...X...............XXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX...........X..X
..X......XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.
XXXXXXXXX..........X...XXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXX...................XXXXXXXX
..X.......XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX....X...
XXXXXXXXXX...............XXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXX.......XXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX................XX
..X.............XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..
XXXXXXXXXXXXXXXX.........XXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXX.............XXXXXXXXXXXX
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX............
X..............XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXX..............XXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXX..............XXXXXXX
.....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.......
XXXXX...........XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXX...............XXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.........XXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.......XXX
....XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX...
XXXX..........XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXX...............XXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXX..........XXXXXXXXXXX
...XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX...........
XXX...........XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXX...............XXXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXX...........XXXXXXXXXX
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX..........
